pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/ElasticTestUtil.java

```diff
@@ -234,13 +236,56 @@ public static Profile addParticipantProfileFromFile(String esIndex, String fileN
         }
     }
 
+    public static Dsm addParticipantDsmFromFile(String esIndex, String fileName, String ddpParticipantId, String dob) {
```

Comment:
How does `participantDsm` differ from just `participant`?  If this is adding a participant to the dsm document, it might be better conveyed as `addParticipantToDsmDoc/Index`.

Reply:
It will add the participant info that is kept in the `DSM` object in the ES doc.